test/tools/yulInterpreter/Interpreter.cpp

```diff
@@ -47,13 +47,11 @@ void InterpreterState::dumpTraceAndState(ostream& _out) const
 	for (auto const& line: trace)
 		_out << "  " << line << endl;
 	_out << "Memory dump:\n";
-	for (size_t i = 0; i < memory.size(); i += 0x20)
-	{
-		bytesConstRef data(memory.data() + i, 0x20);
-		if (boost::algorithm::all_of_equal(data, 0))
-			continue;
-		_out << "  " << std::hex << std::setw(4) << i << ": " << toHex(data.toBytes()) << endl;
-	}
+	map<u256, u256> words;
+	for (auto const& [offset, value]: memory)
+	  words[offset / 0x20] |= value << (256 - 8 - 8 * size_t(offset % 0x20));
+	for (auto const& [offset, value]: words)
+		_out << "  " << std::hex << std::setw(4) << offset << ": " << h256(value).hex() << endl;
```

Comment:
I forgot in my suggestion: this should probably be
```suggestion
		if (value)
			_out << "  " << std::hex << std::setw(4) << offset << ": " << h256(value).hex() << endl;
```